### PR TITLE
Issue312 fetch users out of current room

### PR DESCRIPTION
**Pull request by bitfriend**

### CODE REVIEW

File: app/lib/controllers/chat_room_controller.dart [hunk 1]
```diff
@@ -103,22 +103,22 @@ class ChatRoomController extends GetxController {
...
-      _stream = _currentRoom!.timeline();
+      _stream = convoRoom.timeline();
       // i am fetching messages from remote
       if (_currentRoom == null) {
         // user may close chat screen before long loading completed
         isLoading.value = false;
         return;
       }
-      var msgs = await _stream!.paginateBackwards(10);
+      var msgs = await _stream!.paginateBackwards(5);
```
Comment (gtalha07) [inline app/lib/controllers/chat_room_controller.dart:121]: Why 5? I think 10 is optimal number for fetch. Besides, scrolling is disabled if small number of messages are there initially. Weird though.

**Review by gtalha07 — APPROVED** [review]
LGTM!

File: app/lib/controllers/chat_room_controller.dart (hunk 1 above)
Comment (bitfriend) [reply app/lib/controllers/chat_room_controller.dart:121]: 10 is too large to fetch successful decrypted messages under encryption module.
So I was testing with 5.

Comment (bitfriend) [reply app/lib/controllers/chat_room_controller.dart]: Reverted to 10

**Pull request merged**